### DISCUSSION TITLE
求教是什么原因呢？

### DISCUSSION THREAD

**Discussion by MreDiaos** [zzzgydi/clash-verge#143] [Q&A] [answered]
> 一样的配置我在clash meta for android上没有问题。
> 在clash-verge上Proxy-Groups里只有第一组Proxy这个组的测速是正常的，其他的 分组测速全部是error。

> 请大佬指点迷津。![Screenshot_20220804-114707_NMM.jpg](https://user-images.githubusercontent.com/94931475/182758882-05abbc15-2b01-44d3-9810-c648e54e0821.jpg)

**Comment by zzzgydi** [comment, marked as answer]
由proxy provider use的组，测速都是error的，因为用的测试接口不是一样的

**Reply by MreDiaos** [reply]
> 这个有办法解决么？大佬
